.golangci.yml

```diff
@@ -9,6 +9,9 @@ run:
   # Default: 1m
   timeout: 3m
 
+  skip-files:
+  - example_test.go
```

Comment:
what exactly do we break in this file? Maybe `// nolint` would be a better fit?

Reply:
Good point, I fixed a bunch of issues in `example_test` - only needed `// nolint` in one place 😉